lib/YuiRestClient/Widget/SingleItemSelector.pm

```diff
@@ -0,0 +1,32 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package YuiRestClient::Widget::SingleItemSelector;
```

Comment:
I wonder if we need separate class for `SingleItemSelector`, or for the starters we can try with just `ItemSelector`. We could simply return array of selected items. We can also keep it for the future to inherit from common class once we have `MultiItemSelector`.